source/guides/core-concepts/writing-and-organizing-tests.md

```diff
@@ -318,3 +319,61 @@ it('can subtract numbers', function() {
   assert.equal(subtract(5, 12), -7, 'these numbers are equal')
 })
 ```
+
```

Comment:
Really not sure that this is the best place in the documentation for documenting watching. I plumped for here in the end because the file-watching feeds directly into a productive experience _while writing tests._ Happy to move this elsewhere per your direction @jennifer-shehane .

Reply:
Yeah, I don't think Core Concepts is the right place for this. I think Guides is a better place for it.